meditate/static/js/order/paypal.js

```diff
@@ -0,0 +1,57 @@
+function recordPayPalPayment(paymentId, amount) {
+            var url = '{% url "paypal_charge" %}';
```

Comment:
Template tags won't get substituted in static files.  Either leave as part of .html template, pass the value(s) when the function is called, or include in template with {% include ... %} tag.